packages/components/components/elvis-autocomplete/package.json

```diff
@@ -19,10 +19,11 @@
     "@elvia/elvis-assets-icons": "^3.10.0",
     "@elvia/elvis-colors": "^4.4.1",
     "@elvia/elvis-component-wrapper": "^4.2.0",
-    "@elvia/elvis-toolbox": "^11.5.0",
-    "@elvia/elvis-tooltip": "^1.3.3",
-    "@elvia/elvis-typography": "^2.7.0",
-    "styled-components": "^5.3.3"
+    "@elvia/elvis-toolbox": "^12.0.0",
+    "@elvia/elvis-tooltip": "^1.3.5",
+    "@elvia/elvis-typography": "^2.7.1",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5"
```

Comment:
Their [_For Library Authors_ article](https://emotion.sh/docs/for-library-authors) states

> If you do choose to use Emotion in your library, it is best to list the Emotion packages as peer dependencies in your package.json. This ensures that your library and the consuming application get the same instance of each Emotion package.

Is this something we should consider?

Reply:
Hmm yes, it's not a bad idea, however that does require our users to install @emotion themselves 🤔

Reply:
This also makes me wonder if we should put `@Emotion` in the `ignoreDeps` array in `renovate.json` as all components would have to be updated at the same time on our side, but also on the user's side

Reply:
It may be better to have `@emotion` in peerDeps and also `ignoreDeps` yeah. 
Maybe we should have a little discussion regarding this, so we all agree.

Reply:
Conclusion: We're keeping it as it is. If it creates problems down the road, we can reconsider marking Emotion as peer dependency